### ISSUE TITLE
Using tiled surface data

### ISSUE THREAD

**Issue by GeorgeGayno-NOAA** [ufs-community/ufs-weather-model#1616]
The surface cycling component of the model uses the CCPP sfcsub.F module. That module was updated to optionally use the surface data on tiles in NetCDF. The tiled data is available for all resolutions in the 'fixed' directory. For example, on WCOSS2:

`/lfs/h2/emc/global/noscrub/emc.global/FIX/fix/orog/20220805/C768.mx025/fix_sfc`

The chgres_cube program uses the tiled data to create its coldstart files. For consistency, the model should use the tiled data as well (instead of the GRIB1 gaussian and lat-lon data, which must be horizontally interpolated to the fv3 grid on the fly).

Choosing the tiled data is done through a namelist option. 

This issue should be included as part of the GFS v17 upgrade.

**Comment by yangfanglin** [comment]
@aerorahul The option was added to exglobal_forecast.sh to use fix fields on the tiles a while back but I cannot recall if the option was made consistently in all workflow scripts.    My understanding is that not all "fix" fields are made available on the tiles.  @GeorgeGayno-NOAA is this still correct ?   Nevertheless, my suggestion is to remove the options to use the "old" gaussian and lat-lon data wherever possible for all future UFS applications.

**Comment by GeorgeGayno-NOAA** [comment]
> @aerorahul The option was added to exglobal_forecast.sh to use fix fields on the tiles a while back but I cannot recall if the option was made consistently in all workflow scripts. My understanding is that not all "fix" fields are made available on the tiles. @GeorgeGayno-NOAA is this still correct ? Nevertheless, my suggestion is to remove the options to use the "old" gaussian and lat-lon data wherever possible for all future UFS applications.

@yangfanglin Correct. Not all surface fixed files are available on tiles. For example, the soil moisture climatology is on a gaussian grid - ./fix_am/global_soilmgldas.statsgo.t1534.3072.1536.grb. At some point, I would expect the land team to create an updated dataset on tiles.

**Comment by yangfanglin** [comment]
@junwang-noaa Jun, all RT tests should be switched to use the fix files on the tiles if this has not been done already

**Comment by junwang-noaa** [comment]
@yangfanglin Only some of the global RT tests are using fixed files,  do you mean all the global RT tests or RT tests including global, regional and hafs? The regional and hafs test are still using global lat/lon grib files.

**Comment by yangfanglin** [comment]
@junwang-noaa Jun, since RRFS and online-CMAQ are all running on a single tile, in my opinion they all should be using the fix files created specially for the tile.  I understand HAFS domains may change depending on where the storms are, especially for the moving nests, so global dataset may have to be used for HAFS.   Nevertheless, it is up to each app leads to decide.  

 I'd recommend at least all global RT tests switch to the fix files on the tile.

**Comment by gspetro-NOAA** [comment]
@junwang-noaa Has this issue been resolved?

**Comment by junwang-noaa** [comment]
Not yet.
